docs/index.rst

```diff
@@ -1808,6 +1809,174 @@ logistic function as well as the configuration parameters described in
 the `developers section <https://bigml.com/api/logisticregressions>`_ .
 
 
+Linear Regressions
+------------------
+
+A linear regression is a supervised machine learning method for
+solving regression problems. The implementation is a multiple linear regression
```

Comment:
hmm, saying that the implementation of a linear regression is a multiple linear regression sound a bit circular :) perhaps a "multi-linear approximation"?

Reply:
yes, probably, but that's what the API docs say:

A linear regression is a supervised machine learning method that implements multiple linear regression for solving regression problems.